docs/index.md

```diff
@@ -12,6 +12,7 @@ Below, you'll find the following sections:
   - [Chaincode developer guide](#chaincode-developer-guide)
   - [API developer guide](#api-developer-guide)
 - [Operations guide](#operations-guide)
+- [Continuous Integration](https://github.com/hyperledger/fabric/blob/master/TravisCI_Readme.md)
```

Comment:
@rameshthoomu Why are we bringing this back into the docs? Travis will soon be EOL'ed and it is really irrelevant to anyone but the CI team. I explicitly removed it from the index previously because I did not think it belonged in the end user/developer docs, and at the end of the day, that is what this is really supposed to be.